docs/detections/rules-ui-create.asciidoc

```diff
@@ -261,37 +261,104 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 
 [discrete]
 [[create-esql-rule]]
-=== Create an ES|QL rule
+=== Create an {esql} rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
-. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
+Use {ref}/esql.html[{esql}] to query your source events and aggregate event data. Query results are returned in a table with rows and columns. Each row becomes an alert.
+
+To create an {esql} rule:
 
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page appears.
+. Select **{esql}**, then write a <<esql-rule-query-types,query>>. 
++
+TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help icon,20,20]) to open the in-product reference documentation for all {esql} commands and functions.
 +
-[esql]
+NOTE: To avoid conflicts between the {ref}/esql-commands.html#esql-limit[`LIMIT` value] (which defines how many rows are returned in the results table) and the <<opt-fields-all,`max_signals`>> value (which determines the max number of alerts that can be generated per rule execution), make sure the `max_signals` value does not exceed the `LIMIT` value. If you don't define a `LIMIT` value, the rule automatically uses the `max_signals` value. 
```

Comment:
@vitaliidm let me know if this note needs any tweaks. Thanks in advance for the feedback!